csrankings-j.csv

```diff
@@ -935,6 +935,7 @@ Jian Li 0015,Tsinghua University,http://iiis.tsinghua.edu.cn/~jianli,khvIZWkAAAA
 Jian Li 0021,Shanghai Jiao Tong University,https://sdic.sjtu.edu.cn/index.php/jian-li/,NOSCHOLARPAGE
 Jian Liu 0001,University of Tennessee,http://web.eecs.utk.edu/~jliu,Ckfl4_AAAAAJ
 Jian Liu 0012,Zhejiang University,https://person.zju.edu.cn/0019840,-T_UNHwAAAAJ
+Jian Lou 0001,Zhejiang University,https://sites.google.com/view/jianlou,NOSCHOLARPAGE

```

Comment:
Web page says "researcher", not faculty member; research faculty members are not eligible for inclusion.